quinn-proto/src/transport_parameters.rs

```diff
@@ -176,20 +193,20 @@ impl PreferredAddress {
     }
 }
 
+/// Errors encountered while decoding `TransportParameters`
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Error)]
 pub enum Error {
```

Comment:
```suggestion
pub(crate) enum Error {
```

Reply:
Actually I don't think this is allowed since `Error` appears in the `read()` result type.

Reply:
Whoops, good catch.